main.go

```diff
@@ -613,7 +619,7 @@ func configureVpp() (err error) {
 			Gw:        route.Gw,
 		})
 		if err != nil {
-			return errors.Wrap(err, "cannot add route in vpp")
+			log.Errorf("cannot add route in vpp: %v", err)
```

Comment:
Never sure if failing / continuing is better... 
I think here for debugging sake it's better to not fail too fast.

Reply:
Right, for now it's either that, or we need to be more careful not to add a route that will already be set with VPP, or we need vpplink to not fail when adding a route that already exists. This is the fastest way forward for now...